include/util/guidance/turn_lanes.hpp

```diff
@@ -69,7 +69,7 @@ class LaneTuple
     bool operator!=(const LaneTuple other) const;
 
     LaneID lanes_in_turn;
-    LaneID first_lane_from_the_right;
+    LaneID first_lane_from_the_right; // is INVALID_LANEID when no lanes present
```

Comment:
Would be great to have optionals for all of these - yes.
Since this is exactly the use-case optionals were designed for.